Source/Core/Core/IOS/USB/Bluetooth/WiimoteDevice.h

```diff
@@ -23,98 +24,135 @@ class BluetoothEmu;
 class WiimoteDevice
 {
 public:
-  WiimoteDevice(Device::BluetoothEmu* host, int number, bdaddr_t bd, bool ready = false);
+  using ClassType = std::array<u8, HCI_CLASS_SIZE>;
+  using FeaturesType = std::array<u8, HCI_FEATURES_SIZE>;
+  using LinkKeyType = std::array<u8, HCI_KEY_SIZE>;
+
+  WiimoteDevice(Device::BluetoothEmu* host, int number, bdaddr_t bd);
+
+  WiimoteDevice(const WiimoteDevice&) = delete;
```

Comment:
Minor: rule of 5?

Reply:
I'll default the destructor if if makes you happy. :P